packages/SwingSet/src/kernel/kernel.js

```diff
@@ -576,51 +687,53 @@ export default function buildKernel(
     vatKeeper.setSourceAndOptions(source, options);
     vatKeeper.initializeReapCountdown(options.reapInterval);
 
-    function sendNewVatCallback(results, slots) {
-      const methargs = {
-        body: JSON.stringify(['newVatCallback', [vatID, results]]),
+    function makeVAMessage(arg, slots) {
```

Comment:
This function could do with a better name.

Reply:
yeah, I was trying to keep it short enough to stop prettier from going newline-mad, I'll look for a better one

Reply:
changed to `makeVatAdminMessage`